packages/neon_framework/lib/src/bloc/bloc.dart

```diff
@@ -63,9 +66,13 @@ abstract class InteractiveBloc extends Bloc {
       }
 
       await (refresh ?? this.refresh)();
-    } catch (error, stacktrace) {
-      debugPrint(error.toString());
-      debugPrint(stacktrace.toString());
+    } on Exception catch (error, stackTrace) {
+      _log.info(
+        'An Exception has occurred executing an action.',
+        error,
+        stackTrace,
+      );
```

Comment:
Instead of using it's own logger the method should get the logger passed. Otherwise the messages are really not helpful and don't make a lot of sense.

Reply:
Not resolved?

Reply:
Oh it looks like you fixed it, but added it to the wrong commit.

Reply:
:woman_facepalming: